examples/async_example.py

```diff
@@ -15,13 +15,24 @@
 
 import time
 from rev_ai import apiclient
+from rev_ai.models import CustomVocabulary
 
 # String containing your access token
 access_token = "your_access_token"
 
 # Create your api client
 client = apiclient.RevAiAPIClient(access_token)
 
+# Construct CustomVocabulary objects using our premade class for ease
+mit_professors = CustomVocabulary(
+    ["Robert Berwick", "Noam Chomsky", "Evelina Fedorenko"]
+)
+# Or manually construct CustomVocabulary as such
+other_scientists = {"phrases": ["Albert Einstein"]}
+# Important: place custom vocabularies in a list before passing them into
+# either of client.submit_job_local_file() or sclient.ubmit_job_url()
+custom_vocabularies = [mit_professors, other_scientists]
```

Comment:
To me, it's unclear why these are passed in as a list. Is each individual vocabulary treated differently?

Reply:
Our [API](https://www.rev.ai/docs#operation/SubmitTranscriptionJob) expects a list of CustomVocabulary instances. And an instance of CustomVocabulary contains a group of phrases. 

A CustomVocabulary instance can therefore contain contextually related custom phrases, and these may then be grouped together into a list in any desired combination to fit a user's needs for a certain transcription job.

Reply:
@nishchalb we made this design decision early on with the idea that we could support combining prebuilt custom-vocabularies in the future. At this point though, this does not appear to be a feature that customers want. This API interface gives us more flexibility in the future.